frontend/src/TimerProvider.tsx

```diff
@@ -51,10 +51,13 @@ const _TimerProvider: React.FunctionComponent<{}> = (props) => {
     if (worker !== null) {
       worker.postMessage({command: 'setTimeout', id, timeout: delay});
     }
+    // console.log("TimerProvider set timeout: " + id);
```

Comment:
we shouldn't commit console logs to production code -- it tends to make everything run slower, and we shouldn't debug in production anyways.

Reply:
got it. Just makes debugging a bit harder:)